backend/modules/transform/services/commands/commands.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+import pandas as pd
+
+from ...domain import commands as domain_commands
+
+logger = logging.getLogger(__name__)
+
+
+def transform(command: domain_commands.TransformData):
+    logger.info("Data transformation start.")
```

Comment:
Please let me know, if we should add some unique identified of a dataset. Otherwise these logs do not bring much benefit in my opinion.

Reply:
Sorry, it's not clear to me.

This log says that transformation has started. It's fine according to me.

Reply:
I thought about more informative log, like 
"Dataset <unique id or name> transformation start". Right now we don't have any way to distinguish them.

For example, in extract , we use Path in logs to tell "Dataset /path/to/csv has invalid format"

Reply:
So no.

Dataset it's only painter for data, it does NOT have an identity.